src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            data-cy="Tab"
+            className={tab === selectedTab && 'is-active'}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (tab !== selectedTab) {
+                  onTabSelected(tab);
+                }
+              }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here